mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksView.kt

```diff
@@ -295,4 +300,33 @@ interface MavericksView : LifecycleOwner {
         onFail: (suspend (Throwable) -> Unit)? = null,
         onSuccess: (suspend (T) -> Unit)? = null
     ) = _internalSF(subscriptionLifecycleOwner, asyncProp, deliveryMode, onFail, onSuccess)
+
+    /**
+     * Subscribes to the given Flow within the coroutine scope of the `subscriptionLifecycleOwner`, starting the flow only when the lifecycle
+     * is started, and executing with the coroutine context of Mavericks' `subscriptionCoroutineContextOverride`. This can be utilized to create
+     * customized subscriptions, for example, to drop the first element in the flow before continuing. This is intended to be used with
+     * `viewmodel.stateflow`.
+     *
+     * @param deliveryMode If [UniqueOnly], when this [MavericksView] goes from a stopped to started lifecycle a value
+     * will only be emitted if the value has changed. This is useful for transient views that should only
+     * be shown once (toasts, poptarts), or logging. Most other views should use [RedeliverOnStart], as when a view is destroyed
+     * and recreated the previous state is necessary to recreate the view.
+     *
+     * Use [uniqueOnly] to automatically create a [UniqueOnly] mode with a unique id for this view.
+     *
+     * @param action supports cooperative cancellation. The previous action will be cancelled if it is not completed before
+     * the next one is emitted.
+     */
+    fun <T : Any> Flow<T>.collectLatest(
```

Comment:
`<T : Any>` means you can't use it on flows with nullable values. 
so it's impossible to use with the following scenario: 
```
stateFlow.map { it.someNullableProperty }.collectLatest { 
    ....
}

```

Is it intentional? 

previously it was ok because each value (including nullable) was wrapped into `MavericksTuple1`

Reply:
yea you wouldn't be able to use it on nullables. everything in the codepath has this assumption. one could still wrap the nullable property into something like MavericksTuple to achieve what is done for you with selectSubscribes right? or is there a usecase you are thinking of that would need this extension to explicitly handle nullable values?

Reply:
>one could still wrap the nullable property into something like MavericksTuple to achieve what is done for you with selectSubscribes right?

MavericksTuple is internal, so it can't be used, but it can be replaced with Optional/listOf() etc. 
Problem here is that usually it complicates the code too much (adding boxing/unboxing it manually, plus missing this extenssion from auto-complete), we had a lot of similar code with RxJava which doesn't support nullable values, so if there is a way to allow nullable values, that would be better. 

>or is there a usecase you are thinking of that would need this extension to explicitly handle nullable values?

I feel it's confusing to allow
```
viewModel.onEach(State::nullableProp) { prop ->
}
```
but do not allow
```
viewModel.stateFlow
   .map { it.nullableProp }
   .drop(1)
   .collectLatest { prop ->
   }
```

and this one is really look weird: 
```
viewModel.stateFlow
   .map { it.nullableProp }
   .drop(1)
   .map { listOf(it) } 
   .collectLatest { propList ->
       val prop = propList.firstOrNull()
       ...
   }
```